pages/services/cassandra/2.6.0-3.11.4/configuration/index.md

```diff
@@ -0,0 +1,281 @@
+
+## Rack-Aware Placement
+
+{{ model.techShortName }}'s "rack"-based fault domain support is automatically enabled when specifying a placement constraint that uses the `@zone` key. For example, you could spread {{ model.techShortName }} nodes across a minimum of three different zones/racks by specifying the constraint `[["@zone", "GROUP_BY", "3"]]`. When a placement constraint specifying `@zone` is used, {{ model.techShortName }} nodes will be automatically configured with `rack`s that match the names of the zones. If no placement constraint referencing `@zone` is configured, all nodes will be configured with a default rack of `rack1`.
+
+## {{ model.techName }} Configuration
+
+{{ model.techName }}'s configuration is configurable via the `{{ model.packageName }}` section of the service schema. Consult the service schema for a complete listing of available configuration.
+
+## Multi-datacenter deployment
+
+To replicate data across data centers, {{ model.techName }} requires that you configure each cluster with the addresses of the seed nodes from every remote cluster. Here's what starting a multi-data-center {{ model.techName }} deployment would look like, running inside of a single DC/OS cluster:
+
+### Launch two {{ model.techShortName }} clusters
+
+1. Launch the first cluster with the default configuration:
+
+```shell
+dcos package install {{ model.packageName }}
+```
+
+2. Create an `options.json` file for the second cluster that specifies a different service name and data center name:
+
+```json
+{
+  "service": {
+    "name": "{{ model.serviceName }}2",
+    "data_center": "dc2"
+  }
+}
+```
+
+3. Launch the second cluster with these custom options:
+```
+dcos package install {{ model.packageName }} --options=<options.json>
+```
+
+### Get the seed node IP addresses
+
+<p class="message--note"><strong>NOTE: </strong>If your {{ model.techShortName }} clusters are not on the same network, you must set up a proxying layer to route traffic.</p>
+
+1. Get the list of seed node addresses for the first cluster:
+
+```shell
+dcos {{ model.packageName }} --name={{ model.serviceName }} endpoints node
+```
+
+Alternatively, you can get this information from the scheduler HTTP API:
+
+```json
+DCOS_AUTH_TOKEN=$(dcos config show core.dcos_acs_token)
+DCOS_URL=$(dcos config show core.dcos_url)
+curl -H "authorization:token=$DCOS_AUTH_TOKEN" $DCOS_URL/service/{{ model.serviceName }}/v1/endpoints/node
+```
+
+Your output will resemble:
+
+```
+{
+  "address": [
+    "10.0.1.236:9042",
+    "10.0.0.119:9042"
+  ],
+  "dns": [
+    "node-0-server.{{ model.serviceName }}.autoip.dcos.thisdcos.directory:9042",
+    "node-1-server.{{ model.serviceName }}.autoip.dcos.thisdcos.directory:9042"
+  ],
+  "vip": "node.{{ model.serviceName }}.l4lb.thisdcos.directory:9042"
+}
+```
+
+Note the IPs in the `address` field.
+
+2. Run the same command for your second {{ model.techShortName }} cluster and note the IPs in the `address` field:
```

Comment:
@viivek46 call out the command here. Replace `same command` with the `command`.

Reply:
@anithamuthu79  commands are already below, and we have used "same command" here becuase it is same as first step.